client/dive-common/components/Viewer.vue

```diff
@@ -77,27 +87,18 @@ export default defineComponent({
     const loadError = ref('');
     const baseMulticamDatasetId = ref(null as string | null);
     const datasetId = toRef(props, 'id');
-    const multiCamList: Ref<string[]> = ref([]);
-    const defaultCamera = ref('');
-    const currentCamera = ref('');
+    const multiCamList: Ref<string[]> = ref(['default']);
+    const defaultCamera = ref('default');
+    const selectedCamera = ref('default');
     const playbackComponent = ref(undefined as Vue | undefined);
     const readonlyState = computed(() => props.readonlyMode || props.revision !== undefined);
-    const mediaController = computed(() => {
-      if (playbackComponent.value) {
-        // TODO: Bug in composition-api types incorrectly organizes the static members of a Vue
-        // instance when using typeof ImageAnnotator, so we can't use the "real" type here
-        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-        // @ts-ignore
-        return playbackComponent.value.mediaController as MediaController;
-      }
-      return {} as MediaController;
-    });
+    const { aggregateController, onResize, clear: mediaControllerClear } = useMediaController();
     const { time, updateTime, initialize: initTime } = useTimeObserver();
-    const imageData = ref([] as FrameImage[]);
+    const imageData = ref({ default: [] } as Record<string, FrameImage[]>);
     const datasetType: Ref<DatasetType> = ref('image-sequence');
     const datasetName = ref('');
     const saveInProgress = ref(false);
-    const videoUrl = ref(undefined as undefined | string);
+    const videoUrl = ref({ default: null } as Record<string, null | string>);
```

Comment:
It's subtle, but there are two empty conditions here when there should only be one.

Either the key is missing, or the key is set to null. So you have to compare against both null and undefined. It would be more strict to choose a single empty condition, which is that if no videourl exists for a camera, its key is not in the map at all, so you only have to compare against undefined.